pkg/v3/ocrtypes.go

```diff
@@ -0,0 +1,39 @@
+package ocrtypes
+
+import (
+	"encoding/json"
+
+	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
+)
+
+type AutomationObservation struct {
```

Comment:
I know we don't always do it, but try to start adding documentation to exported structs and functions that describes what they are or what they do.

In this case you would start with
```
// AutomationObservation ...
type AutomationObservation struct {
```

Go style of commenting structs and functions starts with the name of the struct or function and is followed with whatever else. There are automated documentation builders that collect these comments and build full documentation.